packages/discord.js/src/index.js

```diff
@@ -193,13 +194,26 @@ exports.GuildVerificationLevel = require('discord-api-types/v10').GuildVerificat
 exports.InteractionType = require('discord-api-types/v10').InteractionType;
 exports.InteractionResponseType = require('discord-api-types/v10').InteractionResponseType;
 exports.InviteTargetType = require('discord-api-types/v10').InviteTargetType;
+exports.isJSONEncodable = require('@discordjs/builders').isJSONEncodable;
 exports.Locale = require('discord-api-types/v10').Locale;
 exports.MessageType = require('discord-api-types/v10').MessageType;
 exports.MessageFlags = require('discord-api-types/v10').MessageFlags;
 exports.ModalBuilder = require('@discordjs/builders').ModalBuilder;
 exports.OAuth2Scopes = require('discord-api-types/v10').OAuth2Scopes;
 exports.PermissionFlagsBits = require('discord-api-types/v10').PermissionFlagsBits;
 exports.RESTJSONErrorCodes = require('discord-api-types/v10').RESTJSONErrorCodes;
+exports.SlashCommandAttachmentOption = require('@discordjs/builders').SlashCommandAttachmentOption;
+exports.SlashCommandBuilder = require('@discordjs/builders').SlashCommandBuilder;
+exports.SlashCommandBooleanOption = require('@discordjs/builders').SlashCommandBooleanOption;
+exports.SlashCommandChannelOption = require('@discordjs/builders').SlashCommandChannelOption;
+exports.SlashCommandIntegerOption = require('@discordjs/builders').SlashCommandIntegerOption;
+exports.SlashCommandNumberOption = require('@discordjs/builders').SlashCommandNumberOption;
+exports.SlashCommandMentionableOption = require('@discordjs/builders').SlashCommandMentionableOption;
+exports.SlashCommandRoleOption = require('@discordjs/builders').SlashCommandRoleOption;
+exports.SlashCommandStringOption = require('@discordjs/builders').SlashCommandStringOption;
+exports.SlashCommandSubcommandBuilder = require('@discordjs/builders').SlashCommandSubcommandBuilder;
+exports.SlashCommandSubcommandGroupBuilder = require('@discordjs/builders').SlashCommandSubcommandGroupBuilder;
+exports.SlashCommandUserOption = require('@discordjs/builders').SlashCommandUserOption;
```

Comment:
Instead you could just do `exports = { …, …require(‘@discordjs/builders’) }` ?

Reply:
This would overwrite the exports we do for djs custom classes

Reply:
You didn’t read it all, it doesn’t overwrite those

Reply:
It does. What he mean is djs exports `ActionRowBuilder` so it'll be present in exports. Then when adding djs/builder that `ActionRowBuilder` will override the djs one